apps/src/code-studio/initApp/project.js

```diff
@@ -826,7 +827,18 @@ var projects = module.exports = {
       pathName += '/' + action;
     }
     return pathName;
-  }
+  },
+
+  saveThumbnail(blob) {
```

Comment:
I've added a test for `project.saveThumbnail` -- please take another look, as my sinon-fu is a little shaky.